src/coreComponents/constitutive/docs/CO2Brine.rst

```diff
@@ -1,80 +1,256 @@
 .. _CO2-EOS:
 
 ##################################################################################
-CO2 and brine models 
+CO2-brine model 
 ##################################################################################
 
 
 Summary
-=================================
+=======
 
-The equation-of-state and viscosity of CO2 under both sub- and super-critical conditions are computed as a function of both pressure and temperature, using the empirical equations developed by Span & Wagner (1996) and Fenghour & Wakeman (1998), respectively.
+The CO2-brine model implemented in GEOSX includes two components (CO2 and H2O) that are transported by one or two fluid phases (the gas (:math:`g`) phase and the brine (:math:`\ell`) phase).
```

Comment:
The CO2 can be in supercritical, liquid, or gas phase, right?  maybe we need a better terminology than "gas phase", or add an additional note here?

Reply:
You are right, I removed the terminology "gas phase" from the documentation and I now use "CO2 phase" everywhere.